tests/test_adroll_discovery.py

```diff
@@ -0,0 +1,69 @@
+import tap_tester.connections as connections
+import tap_tester.menagerie   as menagerie
+import tap_tester.runner      as runner
+import os
+import unittest
+
+class TestAdrollDiscovery(unittest.TestCase):
+    def setUp(self):
+        missing_envs = [x for x in [
+            "TAP_ADROLL_REFRESH_TOKEN",
+            "TAP_ADROLL_CLIENT_ID",
+            "TAP_ADROLL_CLIENT_SECRET",
+        ] if os.getenv(x) == None]
+        if len(missing_envs) != 0:
+            raise Exception("Missing environment variables: {}".format(missing_envs))
+
+    def name(self):
+        return "tap_tester_adroll_discovery"
+
+    def get_type(self):
+        return "platform.adroll"
+
+    def get_credentials(self):
+        return {
+            'refresh_token': os.getenv('TAP_ADROLL_REFRESH_TOKEN'),
+            'client_id': os.getenv('TAP_ADROLL_CLIENT_ID'),
+            'client_secret': os.getenv('TAP_ADROLL_CLIENT_SECRET'),
+            'access_token': 'fake'
+        }
+
+    def expected_check_streams(self):
+        return {
+            'advertisables',
+        }
+
+    def tap_name(self):
+        return "tap-adwords"
+
+
+    def get_properties(self):
+        return {
+            'start_date' : '2020-03-01T00:00:00Z'
+        }
+
+    def test_run(self):
+        def preserve_refresh_token(existing_conns, payload):
```

Comment:
Might be something to do later, but since this is copied code from the other test, would be nice to share that code and reuse it instead of duplicating...